contributors/design-proposals/local-ephemeral-storage.md

```diff
@@ -0,0 +1,399 @@
+# Local Ephemeral Storage Resource Management
+Authors: jinxu@ vishh@
+
+Currently Kubernetes does not support storage resource usage guarantee and isolation like compute resources such as CPU and memory. This doc details an effort for improving Storage Resource Management in Kubernetes with focus on
+capacity isolation for local ephemeral storage. 
+
+
+# Goals
+* Support ephemeral storage usage isolation for shared root partition among
+  containers and pods
+* Provide ephemeral storage requirement and garantee for containers and pods
+* Add quota management for ephemeral storage for shared root parition
+* Add limitrange support for ephemeral storage
+
+# Non Goals
+* Provide storage usage isolation for non-shared partitions.
+* Support for I/O isolation using CFS & blkio cgroups.
+  * IOPS isn't safe to be a schedulable resource. IOPS on rotational media is very limited compared to other resources like CPU and Memory. This leads to severe resource stranding.
+  * Blkio cgroup + CFS based I/O isolation doesn't provide deterministic behavior compared to memory and cpu cgroups. Years of experience at Google with Borg has taught that relying on blkio or I/O scheduler isn't suitable for multi-tenancy.
```

Comment:
daft question  - what does CFS stands for? can we perhaps expand it?

Reply:
Completely Fair Scheduler, added it